effekt/jvm/src/test/scala/effekt/StdlibTests.scala

```diff
@@ -76,5 +79,8 @@ class StdlibLLVMTests extends StdlibTests {
     examplesDir / "stdlib" / "list" / "build.effekt",
     examplesDir / "stdlib" / "string" / "strings.effekt",
     examplesDir / "stdlib" / "string" / "unicode.effekt",
+
+    // missing support for top-level constants for ANSI escapes
+    examplesDir / "stdlib" / "test",
```

Comment:
Since top-level definitions #496 are fixed now, this should Just :tm: go through :)

Reply:
```suggestion
```
(after rebuilding)